src/components/Popover/Popover.test.tsx

```diff
@@ -17,6 +17,7 @@ describe('<Popover />', () => {
   });
 
   it('should open Popover with trigger button', () => {
+    // this test is redundant b/c of the snapshot above
```

Comment:
That's fair. Should we remove it? Will this be done in a different PR? Is this a TODO comment?

Reply:
yah left a note for myself that this should be removed and left in PR to confirm. Ticket to clean up is https://czi-tech.atlassian.net/browse/EDS-753 and I will trim the comment now :)